### ISSUE TITLE
Units summary view in listing edit form mock

### ISSUE THREAD

**Issue by abbiefarr** [CityOfDetroit/bloom#298]
Let's add an option for property managers to see either their individual listings or the summary of their listings. If they choose a summary of listings, we want to replace the "Listing Units" table with the unit summary table.

This ticket can be closed once this is available in the UI. Ability to edit the unit summary table is not in scope (and is blocked on other work anyway).
